### ISSUE TITLE
Oracle data asset get_batch DatabaseError: (cx_Oracle.DatabaseError) ORA-14459: missing GLOBAL keyword

### ISSUE THREAD

**Issue by NathanFarmer** [fivetran/great_expectations#2596]
**Describe the bug**
When attempting to load an Oracle table in the first cell of the pre-populated Jupyter Notebook after running `great_expectations suite new` I get the error:
```
DatabaseError: (cx_Oracle.DatabaseError) ORA-14459: missing GLOBAL keyword
[SQL: CREATE TEMPORARY TABLE "ge_tmp_c45cd263" AS SELECT  FROM DUAL 
WHERE ROWNUM <= 1000]
```

**To Reproduce**
```
> great expectations init
...
Would you like to configure a Datasource? [Y/n]: y
...
Great Expectations connected to your database!
...
Would you like to profile new Expectations for a single data asset within your new Datasource? [Y/n]: y

You have selected a datasource that is a SQL database. How would you like to specify the data?
1. Enter a table name and schema
...
Please provide the schema name of the table (this is optional) [db]: <schema name>
Please provide the table name (this is required): <table name>
Name the new Expectation Suite [<schema name>.<table name>.warning]: <expect name>
...
Generating example Expectation Suite...
(cx_Oracle.DatabaseError) ORA-14459: missing GLOBAL keyword
[SQL: CREATE TEMPORARY TABLE "ge_tmp_aa71a2e6" AS SELECT  FROM DUAL 
WHERE ROWNUM <= 1000]
(Background on this error at: http://sqlalche.me/e/13/4xp6)
```
Ok that's weird but I heard profiling is experimental so what if I try this:

```
cd great_expectations
great_expectations suite new
Using v2 (Batch Kwargs) API

You have selected a datasource that is a SQL database. How would you like to specify the data?
1. Enter a table name and schema
...
Please provide the schema name of the table (this is optional) [db]: <schema name>
Please provide the table name (this is required): <table name>
Name the new Expectation Suite [<schema name>.<table name>.warning]: <expect name>
```
This opens a Jupyter Notebook with pre-populated code in first cell:
```
import datetime
import great_expectations as ge
import great_expectations.jupyter_ux
from great_expectations.checkpoint import LegacyCheckpoint
from great_expectations.data_context.types.resource_identifiers import ValidationResultIdentifier

context = ge.data_context.DataContext()

# Feel free to change the name of your suite here. Renaming this will not
# remove the other one.
expectation_suite_name = "<expect name>"
suite = context.get_expectation_suite(expectation_suite_name)
suite.expectations = []

batch_kwargs = {'data_asset_name': '<schema name>.<table name>', 'datasource': 'db', 'limit': 1000, 'schema': '<schema name>', 'table': '<table name>'}
batch = context.get_batch(batch_kwargs, suite)
batch.head()
```
Then when I run this first cell I get the full callback:
```
2021-03-24T19:34:44-0400 - INFO - Great Expectations logging enabled at 20 level by JupyterUX module.
2021-03-24T19:34:46-0400 - INFO - Generating query from table batch_kwargs based on limit and offset
---------------------------------------------------------------------------
DatabaseError                             Traceback (most recent call last)
/opt/anaconda3/envs/fin/lib/python3.9/site-packages/sqlalchemy/engine/base.py in _execute_context(self, dialect, constructor, statement, parameters, *args)
   1275                 if not evt_handled:
-> 1276                     self.dialect.do_execute(
   1277                         cursor, statement, parameters, context

/opt/anaconda3/envs/fin/lib/python3.9/site-packages/sqlalchemy/engine/default.py in do_execute(self, cursor, statement, parameters, context)
    607     def do_execute(self, cursor, statement, parameters, context=None):
--> 608         cursor.execute(statement, parameters)
    609 

DatabaseError: ORA-14459: missing GLOBAL keyword

The above exception was the direct cause of the following exception:

DatabaseError                             Traceback (most recent call last)
<ipython-input-1-77f34319b30e> in <module>
     14 
     15 batch_kwargs = {'data_asset_name': '<schema name>.<table name>', 'datasource': 'db', 'limit': 1000, 'schema': '<schema name>', 'table': '<table name>'}
---> 16 batch = context.get_batch(batch_kwargs, suite)
     17 batch.head()

/opt/anaconda3/envs/fin/lib/python3.9/site-packages/great_expectations/data_context/data_context.py in get_batch(self, arg1, arg2, arg3, **kwargs)
   1419             data_asset_type = arg3
   1420         batch_parameters = kwargs.get("batch_parameters")
-> 1421         return self._get_batch_v2(
   1422             batch_kwargs=batch_kwargs,
   1423             expectation_suite_name=expectation_suite_name,

/opt/anaconda3/envs/fin/lib/python3.9/site-packages/great_expectations/data_context/data_context.py in _get_batch_v2(self, batch_kwargs, expectation_suite_name, data_asset_type, batch_parameters)
   1145             expectation_engine=data_asset_type,
   1146         )
-> 1147         return validator.get_dataset()
   1148 
   1149     def _get_batch_v3(

/opt/anaconda3/envs/fin/lib/python3.9/site-packages/great_expectations/validator/validator.py in get_dataset(self)
   1429             init_kwargs = self.batch.data.get_init_kwargs()
   1430             init_kwargs.update(self.init_kwargs)
-> 1431             return self.expectation_engine(
   1432                 batch_kwargs=self.batch.batch_kwargs,
   1433                 batch_parameters=self.batch.batch_parameters,

/opt/anaconda3/envs/fin/lib/python3.9/site-packages/great_expectations/dataset/sqlalchemy_dataset.py in __init__(self, table_name, engine, connection_string, custom_sql, schema, *args, **kwargs)
    566 
    567         if custom_sql:
--> 568             self.create_temporary_table(
    569                 table_name, custom_sql, schema_name=temp_table_schema_name
    570             )

/opt/anaconda3/envs/fin/lib/python3.9/site-packages/great_expectations/dataset/sqlalchemy_dataset.py in create_temporary_table(self, table_name, custom_sql, schema_name)
   1297             )
   1298 
-> 1299         self.engine.execute(stmt)
   1300 
   1301     def column_reflection_fallback(self):

/opt/anaconda3/envs/fin/lib/python3.9/site-packages/sqlalchemy/engine/base.py in execute(self, statement, *multiparams, **params)
   2233 
   2234         connection = self._contextual_connect(close_with_result=True)
-> 2235         return connection.execute(statement, *multiparams, **params)
   2236 
   2237     def scalar(self, statement, *multiparams, **params):

/opt/anaconda3/envs/fin/lib/python3.9/site-packages/sqlalchemy/engine/base.py in execute(self, object_, *multiparams, **params)
   1001         """
   1002         if isinstance(object_, util.string_types[0]):
-> 1003             return self._execute_text(object_, multiparams, params)
   1004         try:
   1005             meth = object_._execute_on_connection

/opt/anaconda3/envs/fin/lib/python3.9/site-packages/sqlalchemy/engine/base.py in _execute_text(self, statement, multiparams, params)
   1170         dialect = self.dialect
   1171         parameters = _distill_params(multiparams, params)
-> 1172         ret = self._execute_context(
   1173             dialect,
   1174             dialect.execution_ctx_cls._init_statement,

/opt/anaconda3/envs/fin/lib/python3.9/site-packages/sqlalchemy/engine/base.py in _execute_context(self, dialect, constructor, statement, parameters, *args)
   1314 
   1315         except BaseException as e:
-> 1316             self._handle_dbapi_exception(
   1317                 e, statement, parameters, cursor, context
   1318             )

/opt/anaconda3/envs/fin/lib/python3.9/site-packages/sqlalchemy/engine/base.py in _handle_dbapi_exception(self, e, statement, parameters, cursor, context)
   1508                 util.raise_(newraise, with_traceback=exc_info[2], from_=e)
   1509             elif should_wrap:
-> 1510                 util.raise_(
   1511                     sqlalchemy_exception, with_traceback=exc_info[2], from_=e
   1512                 )

/opt/anaconda3/envs/fin/lib/python3.9/site-packages/sqlalchemy/util/compat.py in raise_(***failed resolving arguments***)
    180 
    181         try:
--> 182             raise exception
    183         finally:
    184             # credit to

/opt/anaconda3/envs/fin/lib/python3.9/site-packages/sqlalchemy/engine/base.py in _execute_context(self, dialect, constructor, statement, parameters, *args)
   1274                             break
   1275                 if not evt_handled:
-> 1276                     self.dialect.do_execute(
   1277                         cursor, statement, parameters, context
   1278                     )

/opt/anaconda3/envs/fin/lib/python3.9/site-packages/sqlalchemy/engine/default.py in do_execute(self, cursor, statement, parameters, context)
    606 
    607     def do_execute(self, cursor, statement, parameters, context=None):
--> 608         cursor.execute(statement, parameters)
    609 
    610     def do_execute_no_params(self, cursor, statement, context=None):

DatabaseError: (cx_Oracle.DatabaseError) ORA-14459: missing GLOBAL keyword
[SQL: CREATE TEMPORARY TABLE "ge_tmp_9fc83cd3" AS SELECT  FROM DUAL 
WHERE ROWNUM <= 1000]
(Background on this error at: http://sqlalche.me/e/13/4xp6)
```

**Expected behavior:**
I expected to see the batch.head() printed below the cell.

**Environment:**
 - Operating System: MacOS BigSur 11.2.3
 - Great Expectations Version: 0.13.14
Koo

**Comment by NathanFarmer** [comment]
I submitted PR [#2609](https://github.com/great-expectations/great_expectations/pull/2609) to address this issue.

**Comment by eugmandel** [comment]
@NathanFarmer Thank you for finding this issue and, of course, for the PR! We are still working our way through the recent wave of awesome community PRs, so it might take a while.

**Comment by NathanFarmer** [comment]
This issue has been addressed by merged PR [#2609](https://github.com/great-expectations/great_expectations/pull/2609).
